packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx

```diff
@@ -170,36 +173,52 @@ export const CommandMenu = () => {
     [closeCommandMenu],
   );
 
-  const { loading: isPeopleLoading, records: people } =
-    useSearchRecords<Person>({
-      skip: !isCommandMenuOpened,
-      objectNameSingular: CoreObjectNameSingular.Person,
-      limit: 3,
-      searchInput: deferredCommandMenuSearch ?? undefined,
-    });
+  const {
+    matchesSearchFilterObjectRecordsQueryResult,
+    matchesSearchFilterObjectRecordsLoading: loading,
+  } = useMultiObjectSearch({
+    excludedObjects: [CoreObjectNameSingular.Task, CoreObjectNameSingular.Note],
```

Comment:
maybe we actually want to include tasks?

Reply:
task is also using RICH_TEXT